tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_homescreen_delete_app.py

```diff
@@ -51,3 +51,13 @@ def test_delete_app(self):
         # Check that the app is no longer available
         with self.assertRaises(AssertionError):
             self.apps.launch(self.test_data['name'])
+
+    def tearDown(self):
+        self.marionette.switch_to_frame()
```

Comment:
There's a lot of duplication here. Could we move this to a `GaiaApps.uninstall` method?